app/src/main/java/it/niedermann/owncloud/notes/persistence/NoteSQLiteOpenHelper.java

```diff
@@ -239,6 +257,17 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                 recreateDatabase(db);
             }
         }
+        if (oldVersion < 10) {
+            db.execSQL("ALTER TABLE " + table_notes + " ADD COLUMN " + key_excerpt + " INTEGER NOT NULL DEFAULT ''");
+            Cursor cursor = db.query(table_notes, new String[]{key_id, key_content}, null, null, null, null, null, null);
+            while (cursor.moveToNext()) {
+                ContentValues values = new ContentValues();
+                values.put(key_excerpt, NoteUtil.generateNoteExcerpt(cursor.getString(1)));
+                db.update(table_notes, values, key_id + " = ? ", new String[]{cursor.getString(0)});
+            }
+            cursor.close();
+            DatabaseIndexUtil.createIndex(db, table_notes, key_excerpt);
```

Comment:
I don't think that we need an index for this column, since we don't search for values in it and we don't sort by it. What was the reason for you to add an index?

Reply:
Agree, i think we can dump it.